opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -55,3 +58,45 @@ def __hash__(self):
 
 
 _EMPTY_RESOURCE = Resource({})
+
+
+class ResourceDetector:
+    def __init__(self, crash_on_error=False):
+        self.crash_on_error = crash_on_error
+
+    # pylint: disable=no-self-use
+    def detect(self) -> "Resource":
+        return _EMPTY_RESOURCE
+
+
+class OTELResourceDetector:
+    # pylint: disable=no-self-use
+    def detect(self) -> "Resource":
+        env_resources_items = os.environ.get("OTEL_RESOURCE")
```

Comment:
```suggestion
        env_resources_items = os.environ.get("OT_RESOURCE")
```

We used to use OT everywhere instead of OTEL, feel free to ignore this if the conventions have changed.

Reply:
@james-bebbington Can you comment?

Reply:
Yea it seems like this has changed to OTEL at some point - see https://github.com/open-telemetry/opentelemetry-specification/pull/666

Reply:
I believe we can resolve this for now, it's easy to go back and change it if necessary.